### PR TITLE
content(cms): create Resource "engaging-communities-with-archives-video-as-a-tool-for-activism-advocacy-and-archival-work/index"

### PR DESCRIPTION
**Pull request by DEThorpeDRI**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/posts/engaging-communities-with-archives-video-as-a-tool-for-activism-advocacy-and-archival-work/index.mdx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  - aguiar-laura
+  - gillespie-lynsey
+  - ng-yvonne
+  - miller-elizabeth
+tags:
+  - digital-archives
+  - training-education
+categories:
+  - dariah
+featuredImage: images/tim-mossholder-epbirtdg2hk-unsplash-600.jpg
```
Comment (stefanprobst) [inline content/posts/engaging-communities-with-archives-video-as-a-tool-for-activism-advocacy-and-archival-work/index.mdx:18]: assuming the image is from unsplash - does it have a license?

File: content/posts/engaging-communities-with-archives-video-as-a-tool-for-activism-advocacy-and-archival-work/index.mdx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+abstract: "A recording is of ‘Engaging Communities with Archives: Video as a
+  tool for activism, advocacy, and archival work’, a collaborative event hosted
+  by the Digital Repository of Ireland (DRI) and the Public Record Office of
+  Northern Ireland (PRONI) on 7 Sept 2021."
```
Comment (stefanprobst) [inline content/posts/engaging-communities-with-archives-video-as-a-tool-for-activism-advocacy-and-archival-work/index.mdx]:
```suggestion
abstract: "A recording of 'Engaging Communities with Archives: Video as a
  tool for activism, advocacy, and archival work', a collaborative event hosted
  by the Digital Repository of Ireland (DRI) and the Public Record Office of
  Northern Ireland (PRONI) on 7 Sept 2021."
```

File: content/posts/engaging-communities-with-archives-video-as-a-tool-for-activism-advocacy-and-archival-work/index.mdx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+domain: Social Sciences and Humanities
+targetGroup: Domain researchers
+type: event
+remote:
+  date: 2021-09-07T00:00:00.000Z
+licence: ccby-4.0
+toc: false
+uuid: f7C5PByPgDGIoNQqTxXJa
+---
+This recording is of **‘Engaging Communities with Archives: Video as a tool for activism, advocacy, and archival work’**, a collaborative event hosted by the Digital Repository of Ireland (DRI) and the Public Record Office of Northern Ireland (PRONI) on **7 Sept 2021**.
```
Comment (stefanprobst) [inline content/posts/engaging-communities-with-archives-video-as-a-tool-for-activism-advocacy-and-archival-work/index.mdx]:
```suggestion
This recording is of **'Engaging Communities with Archives: Video as a tool for activism, advocacy, and archival work'**, a collaborative event hosted by the Digital Repository of Ireland (DRI) and the Public Record Office of Northern Ireland (PRONI) on **7 Sept 2021**.
```

File: content/posts/engaging-communities-with-archives-video-as-a-tool-for-activism-advocacy-and-archival-work/index.mdx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+The webinar focuses on participatory projects that aim to train or support community groups in using video to tell personal stories, bring about social change, or archive and preserve activism and advocacy work. The aim of the webinar was to open up an international conversation about how organisations can collaborate with care with community groups to empower people with the knowledge and skills to create and preserve their own stories.
+
+Speakers include:
+
+- **Lynsey Gillespie, Archivist at PRONI**: Lynsey is an archivist at PRONI and the curator for Making the Future, a cross-border cultural programme that aims to empower people to use museum collections and archives to explore the past and create a powerful vision for future change. Lynsey speaks about this project, focussing particularly on the ‘Every Day is a School Day’ initiative, a series of 10 short films created by a group of participants with varying degrees of sight loss as part of Making the Future’s 100 Shared Stories programme.
```
Comment (stefanprobst) [inline content/posts/engaging-communities-with-archives-video-as-a-tool-for-activism-advocacy-and-archival-work/index.mdx]:
```suggestion
- **Lynsey Gillespie, Archivist at PRONI**: Lynsey is an archivist at PRONI and the curator for Making the Future, a cross-border cultural programme that aims to empower people to use museum collections and archives to explore the past and create a powerful vision for future change. Lynsey speaks about this project, focussing particularly on the 'Every Day is a School Day' initiative, a series of 10 short films created by a group of participants with varying degrees of sight loss as part of Making the Future's 100 Shared Stories programme.
```

File: content/posts/engaging-communities-with-archives-video-as-a-tool-for-activism-advocacy-and-archival-work/index.mdx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+categories:
+  - dariah
+featuredImage: images/tim-mossholder-epbirtdg2hk-unsplash-600.jpg
+abstract: "A recording is of ‘Engaging Communities with Archives: Video as a
+  tool for activism, advocacy, and archival work’, a collaborative event hosted
+  by the Digital Repository of Ireland (DRI) and the Public Record Office of
+  Northern Ireland (PRONI) on 7 Sept 2021."
+domain: Social Sciences and Humanities
+targetGroup: Domain researchers
+type: event
```
Comment (stefanprobst) [inline content/posts/engaging-communities-with-archives-video-as-a-tool-for-activism-advocacy-and-archival-work/index.mdx]: this should probably be `webinar-recording`?

(i think we should make it clearer in the CMS that `event` should be used when there are multiple sessions, each with their own content)

File: content/posts/engaging-communities-with-archives-video-as-a-tool-for-activism-advocacy-and-archival-work/index.mdx [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+remote:
+  date: 2021-09-07T00:00:00.000Z
```
Comment (stefanprobst) [inline content/posts/engaging-communities-with-archives-video-as-a-tool-for-activism-advocacy-and-archival-work/index.mdx:24]:
```suggestion
remote:
  date: 2021-09-07T00:00:00.000Z
  publisher: Digital Repository of Ireland, Public Record Office of Northern Ireland
  url: https://doi.org/10.7486/DRI.4168n731n
```

File: content/posts/engaging-communities-with-archives-video-as-a-tool-for-activism-advocacy-and-archival-work/index.mdx (hunk 1 above)
Comment (DEThorpeDRI) [reply content/posts/engaging-communities-with-archives-video-as-a-tool-for-activism-advocacy-and-archival-work/index.mdx:18]: Yes, 'Free to use under the Unsplash Licence': 'https://unsplash.com/license'.

(Photo by Tim Mossholder on Unsplash: https://unsplash.com/photos/bo3SHP58C3g)

File: content/posts/engaging-communities-with-archives-video-as-a-tool-for-activism-advocacy-and-archival-work/index.mdx (hunk 5 above)
Comment (DEThorpeDRI) [reply content/posts/engaging-communities-with-archives-video-as-a-tool-for-activism-advocacy-and-archival-work/index.mdx]: Yes, ok, I will note this for future resources

File: content/posts/engaging-communities-with-archives-video-as-a-tool-for-activism-advocacy-and-archival-work/index.mdx (hunk 1 above)
Comment (stefanprobst) [reply content/posts/engaging-communities-with-archives-video-as-a-tool-for-activism-advocacy-and-archival-work/index.mdx:18]: thanks for checking!

**Comment by VickyGarnett** [comment]
HI Debs - I've made a few changes to this resource.  Mostly minor, just to change the abstract to give users more information about the content of the webinar rather than the details of the webinar, if that makes sense :)  - V

**Pull request merged**